File: docs/examples/python/fetch_jwks.py
import os
import requests

CORE_AUTH_BASE_URL = os.environ.get("CORE_AUTH_BASE_URL", "http://localhost:3000")


def fetch_jwks() -> dict:
    res = requests.get(
        f"{CORE_AUTH_BASE_URL}/.well-known/jwks.json",
        timeout=5,
    )
    res.raise_for_status()
    return res.json()

File: docs/examples/node/fetch-jwks.ts
const CORE_AUTH_BASE_URL = process.env.CORE_AUTH_BASE_URL ?? 'http://localhost:3000';

export async function fetchJwks() {
  const res = await fetch(`${CORE_AUTH_BASE_URL}/.well-known/jwks.json`);

  if (!res.ok) {
    throw new Error(`jwks fetch failed: ${res.status}`);
  }

  return res.json();
}
